frontend/components/campaigns/campaignCharities/CampaignCharityAccordionCard.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Box, Stack, Typography, useMediaQuery, useTheme } from '@mui/material';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import {
+  accordianSx,
+  charityDesktopDescriptionSx,
+  charityDesktopImageSx,
+  charityGraphStackSx,
+  charityLogoSx,
+  charityMobileDescriptionSx,
+  charityMobileImageSx,
+  graphSx,
+  stackSx,
+} from '../../../styles/components/charities/CampaignCharityAccordianCard';
+import { CampaignCharityDonationPublicData } from '../../../types/campaignCharities';
+import SmallCompetingGraph from '../../charts/SmallCompetingGraph';
+
+interface Props {
+  campaignCharity: CampaignCharityDonationPublicData;
+}
+
+const CampaignCharityAccordionCard = ({ campaignCharity }: Props) => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+
+  const primaryDonorDonationData = campaignCharity.primaryDonation;
+  const secondaryDonorDonationData = campaignCharity.secondaryDonation;
+
+  return (
+    <Accordion sx={accordianSx}>
+      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+        <Stack direction="row" justifyContent="center" alignItems="center" sx={stackSx}>
```

Comment:
Can the spacing between all the items in the row be the same?
<img width="1202" alt="Screenshot 2022-11-02 at 2 02 04 PM" src="https://user-images.githubusercontent.com/71375383/199410618-7272ab27-5262-42d3-8589-daadbbc10f1d.png">